interpreter/cling/lib/Interpreter/IncrementalParser.h

```diff
@@ -68,9 +71,6 @@ namespace cling {
     // file ID of the memory buffer
     clang::FileID m_VirtualFileID;
 
-    // CI owns it
```

Comment:
Well, now we double free it because as the thiscomment says, the CI owns the ASTConsumer and also deletes it.